config.go

```diff
@@ -1812,7 +1814,9 @@ func parseRPCParams(cConfig *lncfg.Chain, nodeConfig interface{},
 
 	fmt.Println("Attempting automatic RPC configuration to " + daemonName)
 
-	confFile = filepath.Join(confDir, fmt.Sprintf("%v.conf", confFile))
+	if confFile == "" {
+		confFile = filepath.Join(confDir, fmt.Sprintf("%v.conf", confFileBase))
+	}
```

Comment:
wrap at 80 chars

Reply:
Wrapped, not sure about Go/LND's wrapping conventions so please double-check.